nsv13/code/modules/overmap/weapons.dm

```diff
@@ -195,6 +187,11 @@
 			relay('nsv13/sound/effects/ship/reload.ogg')
 			swap_to(FIRE_MODE_TORPEDO)
 
+/obj/structure/overmap/proc/get_max_firemode()
```

Comment:
signals can probably manage stuff like this better.

Reply:
I'd prefer to run 1 simple compare operation than have to create and manage signal objects in memory for this imho